tests/foreman/api/test_syncplan.py

```diff
@@ -607,8 +607,12 @@ def validate_repo_content(
             except AssertionError:
                 sleep(30)
         else:
-            raise AssertionError(
-                'Repository contains invalid number of content entities')
+            repo = repo.read_json()
```

Comment:
@pondrejk `read_json()` is basically 'return raw json for me', which shouldn't be a use case for nailgun in 99% cases :) And you even described why it's bad - we have a missing field in nailgun, using `read_json()` will hide this issue for us. One of great things nailgun gives us in comparison with raw `requests` lib usage - response validations (as well as populating required fields, setting sub-entities as class attributes and lots more), using 'raw' methods like 'read_json' in most cases eliminates nailgun's benefits.
Please add the missing field to nailgun's entity and use regular `read()` instead.